eZ/Bundle/EzPublishMigrationBundle/Command/LegacyStorage/FixImagesVarDirCommand.php

```diff
@@ -0,0 +1,416 @@
+    {
+        $query = $this->connection->createQueryBuilder();
+        $query
+            ->update('ezimagefile', 'i')
+            ->set('i.filepath', $query->expr()->literal($newFilePath))
+            ->where('i.id = :id')
+            ->setParameter('id', $imageId);
+
+        $query->execute();
+
+        $this->imageAttributes[$contentObjectAttributeId][$oldFilePath] = $newFilePath;
+    }
+
+    protected function updateContentObjectAtributes()
+    {
+        foreach ($this->imageAttributes as $attributeId => $files) {
+            $attributeObjects = $this->getContentObjectAtrributesById($attributeId);
+
+            foreach ($attributeObjects as $attributeObject) {
+                $dom = new DOMDocument('1.0', 'utf-8');
+
+                try {
+                    $dom->loadXML($attributeObject['data_text']);
+                } catch (\RuntimeException $e) {
+                    continue;
+                }
+
+                foreach ($dom->getElementsByTagName('ezimage') as $ezimageNode) {
+                    $oldPath = $ezimageNode->getAttribute('url');
+
+                    if (isset($files[$oldPath])) {
+                        $ezimageNode->setAttribute('url', $files[$oldPath]);
+                        $ezimageNode->setAttribute('dirpath', \dirname($files[$oldPath]));
+                    }
+
+                    foreach ($ezimageNode->getElementsByTagName('alias') as $ezimageAlias) {
+                        $oldPath = $ezimageAlias->getAttribute('url');
+                        if (isset($files[$oldPath])) {
+                            $ezimageAlias->setAttribute('url', $files[$oldPath]);
+                            $ezimageAlias->setAttribute('dirpath', \dirname($files[$oldPath]));
+                        }
+                    }
+                }
+
+                $this->updateContentObjectAtribute($attributeObject['id'], $attributeObject['version'], $dom->saveXML());
+            }
+        }
+    }
+
+    /**
+     * @param int $id
+     * @param int $version
+     * @param string $dataText
+     */
+    protected function updateContentObjectAtribute(int $id, int $version, string $dataText)
+    {
+        $query = $this->connection->createQueryBuilder();
+        $query
+            ->update('ezcontentobject_attribute', 'oa')
+            ->set('oa.data_text', $query->expr()->literal($dataText))
+            ->where('oa.id = ' . $id)
+            ->andWhere('oa.version = ' . $version);
+
+        $query->execute();
+    }
+
+    /**
+     * @param int $id
+     *
+     * @return array
+     */
+    protected function getContentObjectAtrributesById(int $id)
+    {
+        $query = $this->connection->createQueryBuilder();
+        $query
+            ->select('oa.data_text, oa.id, oa.version')
+            ->from('ezcontentobject_attribute', 'oa')
+            ->where('oa.id = :id')
+            ->setParameter('id', $id);
```

Comment:
Please provide either `ParameterType::INTEGER` (if already available, I don't remember if that's the case for `6.7`) or `PDO::PARAM_INT`.